src/queue.c

```diff
@@ -861,6 +861,14 @@ gettid(void)
 		if ((f) && tsd->k) ((void(*)(void*))(f))(tsd->k); \
 	} while (0)
 
+#ifdef __ANDROID__
+static void (*_dispatch_thread_detach_callback)(void);
```

Comment:
Global symbol is now a function _dispatch_set_detach_callback as it doesn’t seem possible to express a global var in a silgen. I’ve moved it's declaration into queue_private.h.